apps/researcher/src/app/[locale]/objects/[id]/page.tsx

```diff
@@ -29,7 +30,7 @@ export default async function Details({params}: Props) {
   const object = await heritageObjects.getById({id});
   const locale = useLocale();
   const t = await getTranslations('ObjectDetails');
-  const format = await getFormatter();
+  const formatDate = await getDateFormatter();
```

Comment:
Is it possible to align the name of the variable with the name of the function? I would expect a `getDateFormatter()` to return a `dateFormatter`, not a `formatDate`.

Reply:
This code is removed.